tests/Model/IndexInfoFunctionalTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace MongoDB\Tests\Model;
+
+use MongoDB\Collection;
+use MongoDB\Operation\CreateIndexes;
+use MongoDB\Operation\ListIndexes;
+use MongoDB\Tests\Operation\FunctionalTestCase;
```

Comment:
I'm not keen on using a base test class from another sibling namespace. That said, I changed this locally to extend `MongoDB\Tests\FunctionalTestCase` instead and it passed without issue.